lwreg/helpers.py

```diff
@@ -0,0 +1,69 @@
+from . import utils
+import json
+
+def interactive_config():
```

Comment:
needs a docstring

Reply:
In a bunch of the choices here you indicate what the default is (by putting it in square brackets). I think that's good, but I think just hitting "return" in response to any of the prompts should accept the default. Here you typically generate an error if people just hit return.